### ISSUE TITLE
Incomplete README on how to get started using this lib

### ISSUE THREAD

**Issue by dialex** [chef/inspec-azure#45]
I have set up my inspec profile as you recommended:

```yml
name: acceptance-tests
...
depends:
  - name: azure
    url: https://github.com/chef/inspec-azure/archive/master.tar.gz
```

However when I execute the tests `inspec exec .` I get the error below:

```
/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- ms_rest_azure (LoadError)
  from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
  from /var/lib/gems/2.3.0/gems/inspec-1.33.1/lib/inspec/dsl_shared.rb:14:in `require'
  from libraries/azure_backend.rb:1:in `load_with_context'
  ...
  from /usr/local/bin/inspec:23:in `load'
  from /usr/local/bin/inspec:23:in `<main>'
```

That's because Inspec can't find the required file `ms_rest_azure` on your `azure_backend.rb`. Indeed, **I searched your git repo and there's no file with that name**.

It seems you include some dependencies on your `Gemfile`, but what am I supposed to do? On your [readme](https://github.com/chef/inspec-azure#use-the-resources) you only mention adding the `depends`/`url` to the inspec profile... what am I missing?

**Comment by dialex** [comment]
After I manually downloaded your repo, installed rubocop, and executed bundle install, I was able to run the tests. It seems this only happens on my docker image, and that's because I have to run all gem installs with `sudo`.

Probably InSpec tried to do the exact same thing -- automatically and behind the scenes -- but because it didn't have enough privileges, it failed silently.

**Comment by dialex** [comment]
Nope, that's not it. I fixed the permissions/sudo issue, and still got the same issue. I had to manually download, extract, bundle install your repo for the tests to work.

Shouldn't this be done automatically?

**Comment by chris-rock** [comment]
The latest master version improves the situation. It uses:
- terraform 0.10
- update the gem bundle

@dialex You're right, this will be available in InSpec 2.0 The work is tracked here: https://github.com/chef/inspec/milestone/26 Until then, you need to manually install the following gems along with inspec:
```
gem 'inifile'
gem 'azure_mgmt_compute', '~> 0.11'
gem 'azure_mgmt_resources', '~> 0.11'
gem 'azure_mgmt_network', '~> 0.11'
```

**Comment by dialex** [comment]
Yup, I've added them to a Gemfile, to streamline the whole dependency installation part.
Thanks, looking foward to v2.0 😉
